src/cartesian/getTicks.ts

```diff
@@ -69,7 +71,7 @@ function getTicksStart(
       tickCoord: tailGap > 0 ? tail.coordinate - tailGap * sign : tail.coordinate,
     };
 
-    const isTailShow = isVisible(sign, tail.tickCoord, tailSize, start, end);
+    const isTailShow = isVisible(sign, tail.tickCoord, () => tailSize, start, end);
```

Comment:
Does this even work? Is getTickSize not already evaluated in line 67? 

```
const tailSize = getTickSize(tail, len - 1);
```

Would it not make more sense to define: 
```
const getTailSize = () => getTickSize(tail, len - 1);
const isTailShow = isVisible(sign, tail.tickCoord, getTailSize, start, end);
```

If your solution works, then there is something I am missing. What did I fail to understand? 🤔

Reply:
In this case it doesn't really matter. 

In line 68 we're using the `tailSize`, so we need to compute it anyway. I don't think we can optimize it here.